about.html

```diff
@@ -0,0 +1,134 @@
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=BIZ+UDPGothic:wght@400;700&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="about.css" />
+    <title>Timeline Travel And Tourism Agency</title>
+</head>
+<body>
+    <body>
+    <header>
+        <nav class="navbar">
+            <div class="top-header">
+                <ul class="top-header-navbar">
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"><i class="fa-brands fa-facebook-f"></i></a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li class="nav-item"><a href="#" class="nav-link">English</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">My Page</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Logout</a></li>
+                </ul>
+            </div>
+            <div class="bottom-heading">
+                <a href="#" class="logo"><img src="./images/logo.jpg" alt="logo" />
+                </a>
+                <div class="nav-right">
+                    <ul class="main-navigation">
+                        <li class="nav-item">
+                            <a href="index.html" class="nav-link">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#program" class="nav-link">Program</a>
+                        </li>
+                        <li class="nav-item"><a href="#Join" class="nav-link">Join</a></li>
+                        <li class="nav-item"><a href="#News" class="nav-link">News</a></li>
+                        <li class="nav-item">
+                            <a href="#Team" class="nav-link">Team</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="home.html" class="nav-link campain">CC Campain</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div class="toggleMenu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    <section class="about-main">
+                <p class="intro-p">
+                    "Timeline Travel Agency"
+                </p>
+            <h1 class="desk"> THE WORLD IS A BOOK AND THOSE WHO<br />
+            DO NOT TRAVEL READ ONLY ONE BOOK</h1>
+            <h1 class="mobile">THE WORLD IS A BOOK AND THOSE WHO
+                DO NOT TRAVEL READ ONLY ONE BOOK</h1>
```

Comment:
- [optional] Awesome, please you can wrap all the text with one `h1 `tag to avoid unnecessary repetition. I think you can use the `br` tag to have the text on a new line. Thanks. 😄